plugins/rtmp-services/data/schema/service-schema-v3.json

```
@@ -25,6 +25,24 @@
                         "$ref": "#/definitions/saneUrl",
                         "description": "Link where a logged-in user can find the 'stream key', presented as a button alongside the stream key field."
                     },
+                    "supported video codecs": {
+                        "type": "array",
+                        "description": "Video codecs that are supported by the service.",
+                        "items": {
+                            "type": "string",
+                            "description": "Short-form codec names.",
+                            "minLength": 1
```

Additionally we can specify an enum here of possible strings, that way if someone submits a service with an invalid codec CI/their editor will inform them. Something like

```suggestion
                            "minLength": 1,
                            "enum": [
                                "h264",
                                "hevc"
                            ]
```